server/handlebars/embed/views/main.hbs

```diff
@@ -8,9 +8,9 @@
     {{/if}}
     {{#unless (dataEmpty testPlanReports)}}
         {{#if (isCandidate phase)}}
-            <details id='embed-report-phase-container'>
-                <summary id='candidate-title'><span>Warning! Unapproved Report</span></summary>
-                <div id='candidate-content-container'>
+            <details class="details">
+                <summary><span><h4>Warning! Unapproved Report</h4></span></summary>
```

Comment:
```suggestion
                <summary><h4>Warning! Unapproved Report</h4></summary>
```

Based on my above comment on not needing to target `<span>`. Also typically, inline elements shouldn't contain block elements and I can't say I come across much instances of `<h*>` inside a `<span>` (but I'm sure there are exceptions!)